arches/app/media/css/arches.scss

```diff
@@ -11969,28 +11969,6 @@ a.search-facet-item.disabled {
     color: #777;
 }
 
-.mobile-project-manager-editor div.title-block-title {
-    width: 255px;
-    height: 95px;
-}
-
-.mobile-project-manager-editor .library-card:first-child {
-    margin-top: -5px;
-}
-
-.mobile-project-manager-editor .library-card {
-    margin-left: -5px;
-    background: #fff;
-}
-
-.mobile-project-manager-editor .library-card:hover {
-    background: #fafafa;
-}
-
-.mobile-project-manager-editor .card-nav-container {
-    margin-bottom: 0;
-}
-
 .mpm-project-card {
```

Comment:
`.mpm` was an abbreviation for 'mobile-project-manager', so those classes can probably be removed as well (hopefully they weren't used anywhere else.